rules-tests/EarlyReturn/Rector/If_/RemoveAlwaysElseRector/Fixture/demo_file.php.inc

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Rector\Tests\EarlyReturn\Rector\If_\RemoveAlwaysElseRector\Fixture;
+
+final class DemoFile
+{
+    public function run()
+    { ?>
+        <div>
+            hello
+        </div>
+        <?php
+        if ( rand( 0, 5 ) > 2 ) {
+            return;
+        } else {
+            this is <?php echo "hello"; ?> world
```

Comment:
this is already error even on real code

Reply:
see this https://3v4l.org/t7O0d , already error, so it can't be processed.

Reply:
Ah, yea this eems like PHP in PHP :D

Reply:
The code must be valid PHP :)

The 3v4l link for reference: https://3v4l.org/bP570